### DISCUSSION TITLE
Error with repo

### DISCUSSION THREAD

**Discussion by chrisduk112** [mjmeli/ha-duke-energy-gateway#98] [Q&A] [answered]
Hello

I have setup a fresh HA 8.0 today.  When I go to settings > addons > add new repo I get this error:
![Snag_a1ab12d](https://user-images.githubusercontent.com/37802584/168889760-68c896a7-607c-4f5a-b75b-ddf835033b53.png)

the URL I am entering is: https://github.com/mjmeli/ha-duke-energy-gateway

**Comment by mjmeli** [comment, marked as answer]
You are trying to add the repo under the add-on section. You need to use HACS to add it which is a separate tool you need to install. See https://hacs.xyz/. Once you have HACS, you add this repo under HACS.
